osu.Game/Graphics/Containers/ParallaxContainer.cs

```diff
@@ -71,7 +71,7 @@ protected override void Update()
 
                 double elapsed = MathHelper.Clamp(Clock.ElapsedFrameTime, 0, 1000);
```

Comment:
Tests seem to be failing due to this being unchanged, which sends the interpolated position out of whack:
```suggestion
                 double elapsed = MathHelper.Clamp(Clock.ElapsedFrameTime, 0, 100);
```
Probably best not to apply this outright, but to extract the value to a constant instead.